cmd/agent/common/helpers.go

```diff
@@ -49,3 +53,36 @@ func setupConfig(confFilePath string, configName string, withoutSecrets bool) er
 	}
 	return nil
 }
+
+// SetupSystemProbeConfig adds the system-probe.yaml file to the config object
+func SetupSystemProbeConfig(sysProbeConfFilePath string) error {
+	log.Info(sysProbeConfFilePath)
+	// set the paths where a config file is expected. Without this, if there is a system-probe.yaml
+	// in the default location it will overwrite whatever custom path is passed in
+	if len(sysProbeConfFilePath) != 0 {
+		// If they pass in the file directly, assume we should use it
+		if strings.HasSuffix(sysProbeConfFilePath, ".yaml") {
+			config.Datadog.SetConfigFile(sysProbeConfFilePath)
+		} else {
+			config.Datadog.SetConfigFile(sysProbeConfFilePath + "system-probe.yaml")
+		}
+	} else {
+		// Assume it is in the default location if nothing is passed in
+		config.Datadog.SetConfigName("system-probe")
+		config.Datadog.SetConfigType("yaml")
+		config.Datadog.AddConfigPath(DefaultConfPath)
+	}
+
+	err := config.LoadSystemProbeConfig()
+	if err != nil {
+		config.Datadog.Set("system_probe_config.enabled", false)
```

Comment:
Is the default false? Is this necessary?

Reply:
The default is `nil`, so I thought it might be good to have it set explicitly. The difference in setting it here vs not is below, but I could set a default of `false` in the config.go mapping, which I think would be cleaner potentially. 


If we go with leaving as-is or setting it explicitly here it would be this:

```
if config.Datadog.Get("system_probe_config.enabled") != nil {
// get stats
}
```

vs:

``
if config.Datadog.GetBool("system_probe_config.enabled") = true {
// get stats
}
```